xwiki-platform-core/xwiki-platform-notifications/xwiki-platform-notifications-ui/src/main/resources/XWiki/Notifications/Code/NotificationsAutoWatchPreferencesMacro.xml

```diff
@@ -410,11 +410,11 @@
 {{html clean="false"}}
   &lt;div class="xform"&gt;
     &lt;dl&gt;
-      &lt;dt&gt;&lt;label&gt;$escapetool.xml($services.localization.render('notifications.settings.watch.autowatchmode'))&lt;/label&gt;
-      &lt;p class="xHint"&gt;$escapetool.xml($services.localization.render('notifications.settings.watch.autowatchmode.hint'))&lt;/p&gt;
+      &lt;dt&gt;&lt;label for='preference_notification_autowatch'&gt;$escapetool.xml($services.localization.render('notifications.settings.watch.autowatchmode'))&lt;/label&gt;
+      &lt;p id='preference_notification_autowatch_description' class="xHint"&gt;$escapetool.xml($services.localization.render('notifications.settings.watch.autowatchmode.hint'))&lt;/p&gt;
```

Comment:
Just seeing this again, I'm wondering: do we have any existing uses of this "description" suffix? In most cases we call this "hint" in XWiki, so maybe an id ending with `_hint` would be more consistent.

Reply:
Addressed in https://github.com/xwiki/xwiki-platform/pull/2217/commits/a9e4001ccb02b456060af4bf4d70b414e1bb0ce8